src/group_theory/subgroup.lean

```diff
@@ -404,13 +406,15 @@ end
 @[to_additive] instance fintype_bot : fintype (⊥ : subgroup G) := ⟨{1},
 by {rintro ⟨x, ⟨hx⟩⟩, exact finset.mem_singleton_self _}⟩
 
-@[simp] lemma _root_.add_subgroup.card_bot : fintype.card (⊥ : add_subgroup A) = 1 :=
-fintype.card_eq_one_iff.2
+@[simp] lemma _root_.add_subgroup.card_bot {h : fintype.{v} (⊥ : add_subgroup A)} :
+  @fintype.card (⊥ : add_subgroup A) h = 1 :=
+(@fintype.card_eq_one_iff (⊥ : add_subgroup A) h).2
   ⟨⟨(0 : A), set.mem_singleton 0⟩, λ ⟨y, hy⟩, subtype.eq $ add_subgroup.mem_bot.1 hy⟩
 
 -- `@[to_additive]` doesn't work, because it converts the `1 : ℕ` to `0`.
```

Comment:
there was a `to_additive` update recently - is this still true?

Reply:
No, now it works just fine:
```
@[simp, to_additive] lemma card_bot : fintype.card (⊥ : subgroup G) = 1 :=
fintype.card_eq_one_iff.2
  ⟨⟨(1 : G), set.mem_singleton 1⟩, λ ⟨y, hy⟩, subtype.eq $ subgroup.mem_bot.1 hy⟩
```

Reply:
```suggestion
```

Reply:
I made this change